src/functions.js

```diff
@@ -0,0 +1,39 @@
+function saveToLocalStorage(data) {
+  let tasks = [];
+  if (localStorage.getItem('tasks')) {
+    tasks = JSON.parse(localStorage.getItem('tasks'));
+  }
+
+  tasks.push(data);
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+function deleteList(event) {
+  const button = event.target;
+  const liItem = button.closest('li');
+  const { parentNode } = liItem;
+  parentNode.removeChild(liItem);
+  const listId = liItem.getAttribute('id');
+  const tasks = JSON.parse(localStorage.getItem('tasks'));
+  tasks.splice(listId, 1);
+  for (let i = 0; i < tasks.length; i += 1) {
+    tasks[i].index = i + 1;
+  }
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+  window.location.reload();
+}
```

Comment:
- [ ] Please fix the deleting functionality your deleting functionality is not working so kindly fix it :+1: check below :point_down:  my video for more clarification :slightly_smiling_face: 

[screen-capture (3).webm](https://user-images.githubusercontent.com/45226848/236618261-4f09e766-07ff-41b7-8980-1bbec5e37d48.webm)

- [ ] This is microverse provide [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg&ab_channel=EmiliaAndrzejewska) watch it for more information :+1:

Reply:
@aamirkhan2478 
thank you for your comments, the deleting issue seems fine but the other are the requirements of day 4 project, i have submitted a day 3 project for review .